src/bash.rs

```diff
@@ -402,6 +402,7 @@ pub fn write_completion_script<W: Write>(buffer: &mut W, command: &str, dfa: &DF
     }
 
     write!(buffer, r#"
+    __ltrim_colon_completions "$prefix"
```

Comment:
This turns out to be unnecessary as it only works for colons and `complgen` needs to handle more (e.g. `=`), so the trimming is done manually at https://github.com/adaszko/complgen/blob/b53b7dd8f039a3d5bdc87f2c260f2aa47767992c/src/bash.rs#L327-L328